cmd/runhcs/create-scratch.go

```diff
@@ -34,7 +31,7 @@ var createScratchCommand = cli.Command{
 		if osversion.Get().Build < osversion.RS5 {
```

Comment:
Oh I didn't realize this. We actually should fix this logic. We want to fail `< RS5`. And we should be using this path on the `>= RS5` `else`